applications/CompressiblePotentialFlowApplication/custom_conditions/potential_wall_condition.cpp

```diff
@@ -254,11 +254,11 @@ void PotentialWallCondition<TDim, TNumNodes>::load(Serializer& rSerializer)
 // private operators
 
 template <unsigned int TDim, unsigned int TNumNodes>
-inline Element::Pointer PotentialWallCondition<TDim, TNumNodes>::pGetElement() const
+inline GlobalPointer<Element> PotentialWallCondition<TDim, TNumNodes>::pGetElement() const
 {
-    KRATOS_ERROR_IF(!mpElement.lock())
+    KRATOS_ERROR_IF(mpElement.get() == nullptr)
```

Comment:
```suggestion
    KRATOS_ERROR_IF(mpElement.get())
```

just as information for you, could be changed in a different PR

Reply:
You are missing a _NOT or a ! operator right?
```suggestion
    KRATOS_ERROR_IF_NOT(mpElement.get())
```

Reply:
yes :)